### PR TITLE
Basic hero description group 8

### PR DESCRIPTION
**Pull request by kmalone86**
Eighth group of 5 basic descriptions for heroes.

This only consists of description fields. All number values have been left default and will need to be done later.

hero description
skill name and description
specialty description
These heroes should not be checked off in wiki list as they are not done, just means less work for the person who does the rest of the data.

### CODE REVIEW

File: hero/baal-sezan.json [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+    {
+      "isPassive": false,
+      "soulBurn": 0,
+      "selfSkillBarValue": 0,
+      "soulBurnEffect": "",
+      "awakenUpgrade": true,
+      "cooldown": 0,
+      "name": "Last Requiem",
+      "soulAcquire": 0,
+      "description": "Baal & Sezan attack all enemies with Strike Magic, with a 65% chance to decrease Speed and Defense and make, them unhealable for 2 turns.",
```
Comment (F1l00) [inline hero/baal-sezan.json]: Unnecessary coma after 'make'.
"... Defense and make them ..."

File: hero/baal-sezan.json [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+          ]
+        }
+      ],
+      "buffs": [],
+      "debuffs": []
+    }
+  ],
+  "specialtySkill": {
+    "name": "Necromancer",
+    "description": "Baal & Sezan don't back down easily; something in their hearts just won't let them.\n",
```
Comment (F1l00) [inline hero/baal-sezan.json]: Leftover \n at the end of line.

File: hero/charlotte.json [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+      ],
+      "buffs": [],
+      "debuffs": []
+    },
+    {
+      "isPassive": false,
+      "soulBurn": 0,
+      "selfSkillBarValue": 0,
+      "soulBurnEffect": "Skill cooldown is decreased by 2 turns.",
+      "awakenUpgrade": true,
```
Comment (F1l00) [inline hero/charlotte.json]: false

Comment (F1l00) [inline hero/charlotte.json]: false

File: hero/baal-sezan.json (hunk 1 above)
Comment (kmalone86) [reply hero/baal-sezan.json]: Thanks, that is one of the common pic to text issues, google likes to put "," or "'" at the end of lines sometimes.

File: hero/charlotte.json (hunk 3 above)
Comment (kmalone86) [reply hero/charlotte.json]: Fixed, thanks.

File: hero/baal-sezan.json (hunk 2 above)
Comment (kmalone86) [reply hero/baal-sezan.json]: Fixed, thanks.

**Comment by kmalone86** [comment]
Filo reviewed, so going to merge.

**Pull request merged**